doc/y_cable/design_doc.md

```diff
@@ -53,14 +53,32 @@ Vendors can have several implementations/ways to use this concept
 - Mapping could be such that its a dictionary in 2D
 
 
+```python
+
+    {
+         <vendor_name_1>: {
+             <model_name: <module>
+         },
+         <vendor_name_2> : {
+             <model_name>: <module>
+         }
+    }
 ```
-	{<vendor_name_1>: {<model_name: <module>},
-	<vendor_name_2> : {<model_name>:<module>}}
-```
-- For example
-```
-	{"Credo" : {"model1": "y_cable.py"},
- 	 "Amphenol" : {"model_a": "y_cable_a.py", "model_b":"y_cable_b.py"}}
+
+For example
+
+```python
+
+   {
+        "Credo": {
+             "model1": "y_cable.py"
```

Comment:
Add vendor directory in path: `"vendor1/y_cable.py"`

Reply:
fixed